### DISCUSSION TITLE
How about improving the Layers-panel like this?

### DISCUSSION THREAD

**Discussion by mulder3062** [SVG-Edit/svgedit#817] [New feature request]
For other graphic tools, you can intuitively see and reorder the objects contained in a layer.

How about improving the Layers-panel like this?
If it's ok, I'd like to proceed.

![image](https://user-images.githubusercontent.com/6451589/177737298-85b1d5a1-c21e-4870-810f-2b6d564a5207.png)

**Comment by jfhenon** [comment]
That would be extremely useful indeed. You should definitely proceed. Thanks
